### ISSUE TITLE
BUG: Screen turns black and app craches

### ISSUE THREAD

**Issue by EvertVerboven** [a85/POSTMan-Chrome-Extension#4221]
## App Details:
```
Postman for Mac
Version 5.5.3 (5.5.3)
macOS High Sierra 10.13.2 (17C88)
```

## Issue Report:
When starting postman the window UI disappears after a minute or 2. It just turns black. (view screenshot). After the ui dissappears nothing can be done with the app, your have to Force Quit the application.

 1. Did you encounter this recently, or has this bug always been there:
I encountered it from the moment on I started using the native app.

 2. Expected behaviour:
I can use postman, and the UI is there.

 3. Console logs:
View screenshot.

 4. Screenshots
<img width="1282" alt="screen shot 2018-02-21 at 09 20 10" src="https://user-images.githubusercontent.com/22809599/36469713-78be5da0-16e8-11e8-90de-c02793b17d88.png">

### Reproduce
Start postman native for mac, wait a minute. Reproduces every time on my machine.

**Comment by EvertVerboven** [comment]
Also on: High Sierra 10.13.3 (17D102)

**Comment by harryi3t** [comment]
@EvertYipYip We were not able to reproduce this.
Do you have a lot of tabs open?

**Comment by EvertVerboven** [comment]
Yes
Update: After closing all the tabs in that first minute Postman seems to work fine.

**Comment by zerokid** [comment]
mine also having issue
https://imgur.com/a/13uUL
i try to delete and install again same problem

here are the logs
https://imgur.com/a/r9qz1

**Comment by sdnts** [comment]
Hey guys, this is related to the issue in #4128. As [this comment](https://github.com/postmanlabs/postman-app-support/issues/4128#issuecomment-362178326) explains, this is an issue with the underlying framework Postman uses. From @EvertYipYip's experience, this also looks related to performance. We will investigate and try to get this fixed as soon as possible.

Since these two issues are the same, I'm closing this in favor of #4128, please follow that for updates :)

**Comment by velykov** [comment]
WFT, in windows I got black screen after one minute of starting app

**Comment by sivcan** [comment]
@velykov Try this: https://github.com/postmanlabs/postman-app-support/issues/4703#issuecomment-397517447

**Comment by velykov** [comment]
I add system env variable for disable gpu and remove directories Postman in AppData (Local + Roaming) after this install, but my collections is empty now, i will try login again later

**Comment by cyclobster** [comment]
### I fixed my broken postman with this one weird trick...

On High Sierra I was experiencing the same issue and the fixes in #4703 did not help. As a workaround, I opened Postman and immediately opened up `View` > `Developer` > `Show DevTools (Current View)`.

As long as you're able to do that before the app goes black the dev console will freeze before Postman crashes. Then just press play and Postman will continue as normal. It may freeze 4 or 5 more times when you first start, but after that the app works normally.

**Comment by chanikya494** [comment]
Duplicate of #2256 .

1. Finding your Postman.exe file, which for me was in C:\Users<username>\AppData\Local\Postman\app-6.1.3.
2. Right click on Postman.exe
3. Click on Troubleshoot Compatitility
